docker/package/tezos_setup_wizard.py

```diff
@@ -621,18 +635,24 @@ def import_snapshot(self):
 
             valid_choice = True
 
+            import_flag = ""
+            if is_full_snapshot(self.config["snapshot"]):
+                if yes_or_no("Do you want to run an archive node? (y/N) ", "no"):
+                    import_flag = "--reconstruct "
```

Comment:
A small suggestion, something like:
```suggestion
                if yes_or_no("Do you also want to reconstruct all the chain data to run an archive node? (y/N) ", "no"):
                    import_flag = "--reconstruct "
```
might help make more clear that choosing yes here will take a lot more effort :slightly_smiling_face: 

However considering that the default option is sane and anyone confused will _hopefully_ take it, this might not be that important. I'll let you choose.

Reply:
I like your wording better 🙂